filament/src/Renderer.cpp

```diff
@@ -627,13 +658,25 @@ void FRenderer::renderJob(ArenaScope& arena, FView& view) {
 //    auto debug = fg.getBlackboard().get<FrameGraphTexture>("structure");
 //    fg.forwardResource(fgViewRenderTarget, debug ? debug : input);
 
-    fg.forwardResource(fgViewRenderTarget, input);
-
-    fg.present(fgViewRenderTarget);
-
+     //fg.forwardResource(fgViewRenderTarget, input);
+
+     //fg.present(fgViewRenderTarget);
```

Comment:
Don't leave comments. Git history will preserve the old code, comments used for this is just noise